.github/workflows/infrastructure.yml

```diff
@@ -1,23 +1,32 @@
-# Introduction
-# ***************
-# This code snippet is a GitHub Actions workflow file that automates the setup and configuration of the EXIP project's infrastructure using Azure CLI commands.
-# It creates various Azure resources such as resource groups, app service plans, log analytics workspaces, container registries, virtual networks, subnets, VNET peerings,
-# web apps, front doors, private endpoints, DNS configurations, and diagnostic settings. It also sets environment variables and configures the web apps with app settings and logging options.
+# This code snippet is a YAML configuration file for a GitHub Actions workflow. It defines a series of jobs and steps to set up infrastructure, create base
+# infrastructure resources, configure security settings, configure a web app, enable logging and diagnostic settings, and set up health check alerts.
+# The workflow is triggered on pushes to the 'infrastructure' and 'EMS-2898-database-tier' branches. It uses environment variables to define the product, environment,
+# timezone, and target. The jobs are organized into sections for setup, base infrastructure creation, security configuration, web app configuration, logging and diagnostic settings, and health check alerts.
+# Each step in the workflow performs a specific task using the Azure CLI, such as creating resource groups, app service plans, log analytics workspaces, container registries,
+# virtual networks, subnets, VNET peering, databases, web apps, front doors, WAF policies, private endpoints, DNS configurations, and more.
+# The workflow also includes conditional steps based on the target environment, such as different configurations for pre-production and production environments.
+# Overall, this code snippet provides a comprehensive configuration for setting up infrastructure and configuring various resources in an Azure environment using GitHub Actions.
 #
-# Naming conventions
```

Comment:
should this be removed?

Reply:
Good spot - discussed, we should go with this version, which is the latest updated documentation in the `main` and `no-pdf` branches. This is the most recent version: https://github.com/UK-Export-Finance/exip/commit/e2eb580fe610938da969e335185174493c56b737